convert/tests/test_vertex/test_text_wrapper.py

```diff
@@ -0,0 +1,84 @@
+import collections
+import string
+
+from src.vertex_to_bd.badgerdoc_format.plain_text_converter import TextWrapper
+
+
+def test_pop_beginning_whitespaces_text_begin_with_whitespaces():
```

Comment:
I would also add tests string with more that one space char at the beginning

Reply:
Sorry, but this test uses text with several whitespaces at the beginning